src/components/tabController/index.tsx

```diff
@@ -56,16 +65,18 @@ const getScreenWidth = (useSafeArea: boolean) => {
  * @important: On Android, if using react-native-navigation, make sure to wrap your screen with gestureHandlerRootHOC
  * @importantLink: https://kmagiera.github.io/react-native-gesture-handler/docs/getting-started.html#with-wix-react-native-navigation-https-githubcom-wix-react-native-navigation
  */
-function TabController({
-  initialIndex = 0,
-  selectedIndex,
-  asCarousel = false,
-  items,
-  onChangeIndex = _.noop,
-  carouselPageWidth,
-  useSafeArea = false,
-  children
-}: PropsWithChildren<TabControllerProps>) {
+const TabController = React.forwardRef((props: PropsWithChildren<TabControllerProps>, ref: React.Ref<any>) => {
```

Comment:
I think this can resolve the bottom ignores, you decide if it's better:
```
// @ts-expect-error
const TabController: ForwardRefExoticComponent<
  PropsWithoutRef<TabControllerProps> & RefAttributes<TabControllerImperativeMethods>
// eslint-disable-next-line max-len
> & TabControllerStatics = React.forwardRef((props: PropsWithChildren<TabControllerProps>, ref: React.Ref<TabControllerImperativeMethods>) => {
```

Reply:
It doesn't solve the bottom ignores

Reply:
Strange, it does for me. let me know if you want to look at this together

Reply:
Maybe I did something wrong or misunderstood.. 
Do you want to create a (separate) PR with what u meant?

Reply:
It works for me, but I think it breaks the props, so forget about it